.github/workflows/ci.yml

```diff
@@ -2,34 +2,18 @@ name: Continuous Integration
 
 on:
   pull_request:
-  push:
     branches: [master]
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [18.x]
-
     steps:
     - uses: actions/checkout@v4
 
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ matrix.node-version }}
-
-    - name: Get Yarn cache directory
-      id: yarn-cache-dir-path
-      run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+    - uses: ./.github/actions/setup
 
-    - name: Use Yarn cache
-      uses: actions/cache@v4
-      id: yarn-cache
-      with:
-        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-        key: ${{ runner.os }}-yarn-${{ matrix.node-version }}-${{ hashFiles('**/yarn.lock') }}
+    - name: Test packages
+      run: yarn test
 
     - name: Install dependencies
       run: yarn install --prefer-offline --frozen-lockfile
```

Comment:
@carbonrobot should I remove this Install dependencies step here as well, since it appears to be handled in the `actions/setup` workflow?